app/Kconfig

```diff
@@ -317,6 +317,9 @@ config KERNEL_BIN_NAME
 config REBOOT
 	default y
 
+config USB
+	default y if HAS_HW_NRF_USBD
```

Comment:
For our stm32 target, and soon SAMD21 targets, can we also set `y` for `USB_DC_STM32` and `USB_DC_SAM`?

Reply:
@Nicell Ok, can we ditch the SAMD stuff for now, and leave the STM32, until I can dig further? Sooner we merge this, the less conflicts.